### ISSUE TITLE
Test based on subclass of a spec not detected

### ISSUE THREAD

**Issue by xiaodongw** [kotest/kotest#5504]
I have some tests organized by this way:
```kotlin
// parent class for all my integration tests
abstract class ServiceIntegrationSpec : FunSpec() {
  ...
  override fun beforeSpec(spec: Spec) {
    service.start()
  }
}

// the actual test
class XxxIntegrationSpec : ServiceIntegrationSpec() {
  init {
    test("some test") {...}
  }
}
```
I can run the test with Gradle without problem.
When I open the XxxIntegrationSpec in Intellij, it doesn't recognize it as a test (no run menu or icon). It used to work with the old Intellij plugin (kotlin-test).
I'm using kotest plugin 0.90.16, with kotest 4.1.0.259-SNAPSHOT

**Comment by Obirah** [comment]
Same for me with plugin version 1.0.0 and KoTest 4.1.0.RC2 - would really love to be able to get rid of redundancies with abstract specs while still being able to execute single test cases from IntelliJ.

**Comment by sksamuel** [comment]
Fixed in 1.0.1 which is pending jetbrains approval on the plugin store.
